libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -704,7 +703,6 @@ public void takeScreenshot(NavigationSnapshotReadyCallback navigationSnapshotRea
   @SuppressLint("MissingPermission")
   private void initializeLocationComponent(MapView mapView, MapboxMap map) {
     locationComponent = map.getLocationComponent();
-    map.setMinZoomPreference(NAVIGATION_MINIMUM_MAP_ZOOM);
```

Comment:
Unrelated but could you also remove `private static final double NAVIGATION_INITIAL_MAP_ZOOM = 17d;` variable never used?

Reply:
:+1: